src/app/modules/terminal/components/terminal/terminal.component.ts

```diff
@@ -106,17 +106,15 @@ export class TerminalComponent implements OnInit, OnDestroy {
       take(1),
       tap((token) => {
         this.initializeWebShell(token);
-        this.shellService.shellOutput.pipe(untilDestroyed(this)).subscribe(() => {});
+        this.shellService.shellOutput$.pipe(untilDestroyed(this)).subscribe(() => {});
```

Comment:
Can you elaborate why we need this empty subscription please? If it is for the websocket to stay connected, there is a permanent subscription inside the service to `ws$` which would suffice.